packages/client/scripts/default-wasm.js

```diff
@@ -1,7 +1,7 @@
 class PrismaClient {
   constructor() {
     throw new Error(
-      `@prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.
+      `To use @prisma/client in an edge runtime, enable \`driverAdapters\` in your Prisma schema and run \`prisma generate\`.
```

Comment:
What will be the long term message here after the preview feature is removed?

Reply:
It will the same as usual I guess, "Prisma client did not generate, please make sure to run prisma generate"